### PR TITLE
Add information about security (for women).

### PR DESCRIPTION
**Pull request by skd**

### CODE REVIEW

File: conversation_tree.textproto [hunk 1]
```diff
@@ -20,6 +20,12 @@ node {
 
 node {
   name: "Я беженец / беженка"
+  answer { text: "‼️ Важно! Даже в Швейцарии не забывайте следить за свою безопасности! Знакомитесь с нашими советами:" }
```
Comment (crdev) [inline conversation_tree.textproto:23]: "‼️ Важно ‼️ Даже в Швейцарии не забывайте следить за своей безопасностью! Ознакомьтесь с нашими советами:"

File: conversation_tree.textproto [hunk 2]
```diff
@@ -20,6 +20,12 @@ node {
...
+  answer { photo: "safety1.jpg" }
+  answer { photo: "safety2.jpg" }
+  answer { photo: "safety3.jpg" }
+  answer { photo: "safety4.jpg" }
+  answer { photo: "safety5.jpg" }
```
Comment (crdev) [inline conversation_tree.textproto:28]: I think we could also add links represented by QR codes in the last slide, wdyt?

**Review by crdev — APPROVED** [review]

File: conversation_tree.textproto (hunk 1 above)
Comment (skd) [reply conversation_tree.textproto]: Thanks!

File: conversation_tree.textproto (hunk 2 above)
Comment (skd) [reply conversation_tree.textproto:28]: Excellent suggestion. Done!

File: conversation_tree.textproto [hunk 3]
```diff
@@ -20,6 +20,23 @@ node {
 
 node {
   name: "Я беженец / беженка"
+  answer { text: "‼️ Важно ‼️ Даже в Швейцарии не забывайте следить за своей безопасностью! Ознакомьтесь с нашими советами:" }
+  answer { photo: "safety1.jpg" }
+  answer { photo: "safety2.jpg" }
+  answer { photo: "safety3.jpg" }
+  answer { photo: "safety4.jpg" }
+  answer { photo: "safety5.jpg" }
+  answer { links {
```
Comment (crdev) [inline conversation_tree.textproto:29]: Let's perhaps format the file to avoid same-line keys for consistency with the rest of code

**Review by crdev — APPROVED** [review]
Great links, thanks!

**Pull request merged**